src/answers-umd.js

```diff
@@ -428,6 +436,26 @@ class Answers {
       onFinally: config.onFinally || function () {}
     });
   }
+
+  /*
+   * Adds context as a parameter for the query API calls.
+   * @param {Object} context The context object passed in the API calls
+   */
+  setContext (context = {}) {
+    if (context && !isValidContext(JSON.stringify(context))) {
+      return;
```

Comment:
Do we want to console.error here as well?

Reply:
Good catch! Added